Gemfile

```diff
@@ -10,4 +10,6 @@ gem "test-unit", "~> 3.0"
 gem "rack", "< 2.0"
 gem 'minitest', '~> 5.8'
 
+gem "nio"
```

Comment:
Whoops! Thanks.
On Sat, Aug 20, 2016 at 7:57 PM Tony Arcieri <notifications@github.com>
wrote:

> In Gemfile <https://github.com/puma/puma/pull/1056#discussion_r75588178>:
>
> > @@ -10,4 +10,6 @@ gem "test-unit", "~> 3.0"
> >  gem "rack", "< 2.0"
> >  gem 'minitest', '~> 5.8'
> >
> > +gem "nio"
>
> I think you want nio4r here: https://rubygems.org/gems/nio
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/puma/puma/pull/1056/files/d8bf71a9ef09acd25932a4b528812a58bd8ac090#r75588178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAAB1Z6Sro-20ZyMFv7RlgepDc9K2IZks5qh76sgaJpZM4JpNhI>
> .
>